mount/file_test.go

```diff
@@ -0,0 +1,68 @@
+package mount
+
+import (
+	"bytes"
+	"context"
+	"crypto/rand"
+	"io"
+	"io/ioutil"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestFileMount(t *testing.T) {
+	const size = 1024
+
+	// create a temp file.
+	dir := t.TempDir()
+	f, err := ioutil.TempFile(dir, "")
+
```

Comment:
defer f.Close()
defer os.Remove(f)

Reply:
`os.Remove(f)` not needed, as it's already covered by `t.TempDir()`.